Makefile

```diff
@@ -5,7 +5,7 @@ BIN = selftest test
 
 all: $(BIN)
 
-utcp.o: utcp.c utcp.h utcp_priv.h
+utcp.o: utcp.c
```

Comment:
Ähem, was soll das?

Wenn die Header nicht mehr in der Prerequisites-Liste einer .o-Datei stehen, wird die entsprechende .o-Datei auch nicht mehr neu compiliert, wenn sich die .h-Datei ändert. Das hatte also schon seinen Grund, warum die da drinstanden.